python/Ganga/Runtime/bootstrap.py

```diff
@@ -181,6 +181,10 @@ def manualExportToGPI(my_interface=None):
     from Ganga.GPIDev.Lib.Registry.JobRegistry import jobSlice
     exportToInterface(my_interface, "jobSlice", jobSlice, "Functions")
 
+    """Needed for the JobTime if we plan to use the interface to eval the XML as well"""
+    import datetime
+    exportToInterface(my_interface, "datetime", datetime, 'Classes')
```

Comment:
In this case I'll buffer the output from the flat eval which calls a mystery parameter which is badly defined aka the Config system. Using the GPI dictionary this way made life easier as we knew exactly what is misbehaving and means 1 less random dictionary we're looking up against in the future.